tests/s3/versioning/test_versioning_multipart_part_2.py

```diff
@@ -0,0 +1,228 @@
+        self.log.info("Step 4: Upload 4 more versions to mpobject1 and then multipart upload.")
+        for i in range(1, 4):
+            system_utils.create_file(self.test_file_path, count=10 * i)
+            s3ver_cmn_lib.upload_version(self.s3test_obj, self.bucket_name, self.object_name,
+                                         self.test_file_path, versions_dict=versions)
+        s3ver_cmn_lib.upload_version(self.s3mp_test_obj, self.bucket_name, self.object_name,
+                                     self.test_file_path, versions_dict=versions,
+                                     is_multipart=True, total_parts=10, file_size=200)
+        self.log.info("Step 5: PUT Bucket versioning with status as Suspended.")
+        self.s3ver_test_obj.put_bucket_versioning(self.bucket_name, status="Suspended")
+        self.log.info("Step 6: Perform PUT Object to mpobject1 (version with 'versionId=null').")
+        s3ver_cmn_lib.upload_version(self.s3test_obj, self.bucket_name, self.object_name,
+                                     self.test_file_path, versions_dict=versions,
+                                     chk_null_version=True)
+        self.log.info("Step 7: Perform List Object Versions.")
+        s3ver_cmn_lib.check_list_object_versions(self.s3ver_test_obj, self.bucket_name, versions)
+        self.log.info("Step 8: Perform List Objects.")
+        resp = self.s3test_obj.object_list(self.bucket_name)
+        assert_utils.assert_true(resp[0], resp[1])
+        assert_utils.assert_in(self.object_name, resp[1],
+                               f"Failed to list versioned object {self.object_name}")
+        self.log.info("Step 9: Perform GET/HEAD Object.")
+        s3ver_cmn_lib.check_get_head_object_version(self.s3test_obj, self.s3ver_test_obj,
+                                                    self.bucket_name, self.object_name)
+        self.log.info("Step 10: Perform DELETE Object for all 6 versions viz. null, id1..id5.")
+        s3ver_cmn_lib.empty_versioned_bucket(self.s3ver_test_obj, self.bucket_name)
+        self.log.info("Step 11: Perform List Object Versions.")
+        s3ver_cmn_lib.check_list_object_versions(
+            self.s3ver_test_obj, bucket_name=self.bucket_name, expected_versions={})
+        self.log.info("Step 12: Perform List Objects.")
+        resp = self.s3test_obj.object_list(self.bucket_name)
+        assert_utils.assert_true(resp[0], resp[1])
+        assert_utils.assert_not_in(self.object_name, resp[1], f"Listed object {self.object_name}")
+        self.log.info("Step 13: Perform GET/HEAD Object.")
+        s3ver_cmn_lib.check_get_head_object_version(self.s3test_obj, self.s3ver_test_obj,
+                                                    self.bucket_name, self.object_name,
+                                                    get_error_msg=err_msg.NO_SUCH_KEY_ERR,
+                                                    head_error_msg=err_msg.NOT_FOUND_ERR)
```

Comment:
we might need to add param for skip_polling incase failure of getting keys, once Apoorva's PR is merged